### PR TITLE
Add directories to backup

### PR DESCRIPTION
**Pull request by TiJof**
Add users's cron backup and local bin for all users

### CODE REVIEW

File: bin/xivo-backup [hunk 1]
```diff
@@ -75,7 +76,8 @@ case "${BACKUP_TYPE}" in
                  /var/lib/xivo-provd/
                  /var/lib/xivo/
                  /var/log/asterisk/
-                 /var/spool/asterisk/"
+                 /var/spool/asterisk/
+                 /var/spool/cron/crontabs/"
```
Comment (sduthil) [inline bin/xivo-backup]: Could you add /etc/crontab, too?

**Comment by sduthil** [comment]
Could you open another pull request on https://github.com/xivo-pbx/xivo-doc to add the backuped directories in `source/system/backup_restore.rst`, please?

File: bin/xivo-backup (hunk 1 above)
Comment (TiJof) [inline bin/xivo-backup]: Done

**Comment by TiJof** [comment]
Doc is also updated !

**Comment by sduthil** [comment]
Thanks :)

**Pull request merged**